jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/bars/AppBar.java

```diff
@@ -57,6 +57,16 @@ public boolean hasImageFadingOnScroll() {
         return hasClass("v-app-bar--fade-img-on-scroll");
     }
 
+    @JDIAction("Check that '{name}' is on top position")
+    public boolean isOnTopPosition() {
+        return !hasClass("v-app-bar--is-scrolled") || attr("style").contains(BarHeight.THRESHOLD.height);
+    }
+
+    @JDIAction("Check that '{name}' is on top position")
+    public boolean isOnBottomPosition() {
+        return hasClass("v-app-bar--is-scrolled") || attr("style").contains(BarHeight.SHORT.height);
```

Comment:
'v-app-bar--is-scrolled' does not guarantee that it is at the bottom, any position besides 'top' has v-app-bar--is-scrolled

Reply:
agreed
fixed